### DISCUSSION TITLE
Since today Tab Mix Plus could not be verified for use in Firefox and has been disabled

### DISCUSSION THREAD

**Discussion by lgander** [onemen/TabMixPlus#226] [General]
I've been happily using TMP for many months. 
A couple of times it stopped working and had to reinstall, but today no matter what I do it's still not verified/working.

Firefox 115.0b2
Windows 11
tab_mix_plus-dev-build  [v1.0.0-pre.15](https://github.com/onemen/TabMixPlus/releases/tag/v1.0.0-pre.15)

Any ideas?

**Comment by onemen** [comment]
download and install this test build
https://tabmixplus.org/dev-builds/test%20builds/1.0.0-pre.16.20230607.1.xpi

**Reply by lgander** [reply]
Thanks for the quick reply. 
I still got this when trying to install it
![image](https://github.com/onemen/TabMixPlus/assets/39671907/87c5cbd4-2c25-4e8d-ba98-033178af9fb0)

**Comment by onemen** [comment]
check in https://github.com/xiaoxiaoflood/firefox-scripts#instructions, that you have the latest version of all files
also you may have to:
- uninstall Tab mix
- close Firefox
- start Firefox
- install new Tab mix version

**Reply by lgander** [reply]
I've tried downloading both zips and replacing the files.
I've also tried that uninstall-restart-reinstall process.
I still got the same error. 

I really don't know what else can I try.  Do you have any other ideas?

Thanks again!

**Comment by onemen** [comment]
more tips https://github.com/onemen/TabMixPlus/issues/159

**Reply by lgander** [reply]
I've read and tried everything.

I also created a new profile, copied de utils folder inside a "chrome" folder I created inside the profile folder. Same error trying to install the xpi.

Let me copy some screenshots just in case.
![image](https://github.com/onemen/TabMixPlus/assets/39671907/311f758c-94f4-4504-8847-277a4c9e14ec)
![image](https://github.com/onemen/TabMixPlus/assets/39671907/0c320700-a1cc-494d-afd6-76a2d4b19227)
![image](https://github.com/onemen/TabMixPlus/assets/39671907/c1e8b370-ba9a-43a2-bbb2-fffa2e215980)

I'm on windows 11. I'm using XYplorer, that why the explorer looks different. 

It was working for long months, so not sure what could change.
Do you have any other idea? Is there any way to try downgrading FF maybe?

Thanks again!

**Comment by 117649** [comment]
Try set in about:config xpinstall.whitelist.required to false, for some cases it may help.

**Reply by lgander** [reply]
Thanks for the recommendation. 
Still doesnt work.
This is a screenshot of my other xpinstall in case you see something wrong there:
![image](https://github.com/onemen/TabMixPlus/assets/39671907/a26c9afa-d54b-4e0c-a433-5650b4990120)

**Comment by lgander** [comment]
Ive made a backup of my profile.
Completely I uninstalled and reinstalled firefox..
Copied the profile again.
Now its working.

Thanks a lot for taking the time to help me!
